src/udev/udev-varlink.c

```diff
@@ -258,6 +278,7 @@ int udev_open_varlink(Manager *m, int fd) {
         r = varlink_server_bind_method_many(
                         m->varlink_server,
                         "io.systemd.udev.Exit", vl_method_exit,
+                        "io.systemd.udev.GetLogLevel", vl_method_get_log_level,
```

Comment:
as in the other PR, this should probably be generic, i.e. io.systemd.service.GetLogLevel() so that we can implement it in other daemons as well